llvm/test/MC/Sparc/Relocations/relocation-specifier.s

```diff
@@ -1,20 +1,12 @@
 # RUN: llvm-mc %s -triple=sparc | FileCheck %s --check-prefix=ASM
```

Comment:
It's usually easier to maintain to test different ISAs with the same test file.
You can use
```
--check-prefixes=OBJDUMP,OBJDUMP-V8
--check-prefixes=OBJDUMP,OBJDUMP-V9
```